crates/blockifier/src/execution/contract_class.rs

```diff
@@ -130,7 +130,7 @@ impl ContractClassV0 {
             + self.n_builtins()
             + self.bytecode_length()
             + 1; // Hinted class hash.
-        // The hashed data size is approximately the number of hashes (invoked in hash chains).
+                 // The hashed data size is approximately the number of hashes (invoked in hash chains).
```

Comment:
you have a few extra spaces here:
```suggestion
        // The hashed data size is approximately the number of hashes (invoked in hash chains).
```